lib/src/module.dart

```diff
@@ -142,6 +142,8 @@ abstract class Module {
     } else if (reserveDefinitionName &&
         !Sanitizer.isSanitary(definitionName!)) {
       throw InvalidReservedNameException();
+    } else if (definitionName == '') {
+      throw NullReservedNameException();
```

Comment:
Shouldn't this be thrown `reserveDefinitionName` is set to true?  If it's not, we should probably silently rename it to something safe like some auto-generated name?